github/Issue.py

```diff
@@ -312,16 +313,21 @@ def get_comment(self, id):
         )
         return github.IssueComment.IssueComment(self._requester, headers, data, completed=True)
 
-    def get_comments(self):
+    def get_comments(self, since=github.GithubObject.NotSet):
```

Comment:
I don't understand the benefit of using `github.GithubObject.NotSet` over `None`. It seems like `github.GithubObject.NotSet` is a special case each time `since` is referenced